src/main/java/co/rsk/bitcoinj/script/Script.java

```diff
@@ -97,6 +97,9 @@ public enum VerifyFlag {
 
     private static final Logger log = LoggerFactory.getLogger(Script.class);
     public static final long MAX_SCRIPT_ELEMENT_SIZE = 520;  // bytes
+
+    /** The maximum size in bytes of a standard witnessScript */
+    public static final long MAX_STANDARD_P2WSH_SCRIPT_SIZE = 3600;
```

Comment:
This value is not being used. Where is it supposed to be used, `rskj`?

Reply:
yes